turbomoleio/core/control.py

```diff
@@ -166,22 +166,22 @@ class Gradient(MSONable):
     """
 
     def __init__(self, gradients=None, energies=None, molecules=None,
-                 periodicity='0D', lattice_vectors=None, lattice_gradients=None):
+                 periodicity=0, lattice_vectors=None, lattice_gradients=None):
         """
         Args:
             gradients (list): gradients for all the steps and all the atoms
                 with shape nsteps*natoms*3.
             energies (list): total energies for each step.
             molecules (list): list of MoleculeSystem with coordinates for each step.
```

Comment:
this is an option but I'm afraid breaking backward compatibility would be a problem for basf. Maybe we can add a structures argument (default to None) so that it will not break it ? We can also add a systems property that returns either molecules or structures depending on which one is presentL

Reply:
I feel like adding several arguments may make the things more complicated. Maybe we could agree with BASF for the realease a backward incompatible version at a later stage? I suppose there are several places in the code where `molecule` is used and could be replaced by a common term. I am not really convinced by `system` as this could be interpreted as a reference to os.system() and cause confusion.

Reply:
I propose we leave it this way for the moment. Another possibility is to indeed break backward compatibility somehow but allow to deserialize the old version (should be doable using either the "@version" or just checking whether molecules is there in the dictionary or not). One could even have a molecules property that returns the systems when they are a molecule so that scripts using the gradient object would continue to work. What do you think ?

Reply:
ok